superset-frontend/src/dashboard/components/SliceHeaderControls/index.jsx

```diff
@@ -266,6 +266,7 @@ class SliceHeaderControls extends React.PureComponent {
             copyMenuItemTitle={t('Copy chart URL')}
             emailMenuItemTitle={t('Share chart by email')}
             emailSubject={t('Superset chart')}
+            emailBody={t('Check out this chart: ')}
```

Comment:
Has anyone provided the copy? I am just wondering how do we go about copy in general when required. CC @rusackas

Reply:
We don't have official copy-writing practices, per se. What we can do in instances like this one is consider (and potentially approve) the PR, and we could add the `need:design-review` label, which means that @mihir174 and/or @steejay will take a look from a _design_ perspective. If a body of copy gets large enough or seems contentious, we could take it to the dev@ mailing list for lazy consensus. 

This case seems small enough though that we could just merge it, and if anyone has better ideas, they can open a new PR to tweak it. This fixes the bug, in the meantime.